catroid/src/test/java/org/catrobat/catroid/test/visualplacement/VisualPlacementTouchListenerTest.java

```diff
@@ -56,9 +58,17 @@
 	@Mock
 	CoordinateInterface coordinateInterface;
 
+	@Mock
+	TimeProvider timeProvider;
+
 	@InjectMocks
 	VisualPlacementTouchListener listener;
 
+	@Before
+	public void setUp() {
+		when(timeProvider.elapsedRealtime()).thenReturn((long) 0);
```

Comment:
```suggestion
		when(firstEvent.getDownTime()).thenReturn((long) 0);
```
I am not sure whether that's the right MotionEvent here. Please correct if it is incorrect for setUp().